Modules/Bridge/VTK/include/itkVTKImageExport.h

```diff
@@ -64,55 +64,67 @@ class ITK_TEMPLATE_EXPORT VTKImageExport:public VTKImageExportBase
   static constexpr unsigned int InputImageDimension = TInputImage::ImageDimension;
 
   /** Run-time type information (and related methods). */
-  itkTypeMacro(VTKImageExport, VTKImageExportBase);
+  itkTypeMacro( VTKImageExport, VTKImageExportBase );
 
   /** Method for creation through the object factory. */
-  itkNewMacro(Self);
+  itkNewMacro( Self );
 
   /** The type of the input image. */
   using InputImageType = TInputImage;
 
 #ifdef ITK_USE_CONCEPT_CHECKING
-  itkConceptMacro( ImageDimensionCheck,
-                   ( Concept::SameDimensionOrMinusOneOrTwo<
-                     3,Self::InputImageDimension > ) );
+  itkConceptMacro( ImageDimensionCheck, (Concept::SameDimensionOrMinusOneOrTwo< 3, Self::InputImageDimension >));
 #endif
   /** Set the input image of this image exporter. */
   using Superclass::SetInput;
-  void SetInput(const InputImageType *);
-  InputImageType * GetInput();
+  void
+  SetInput( const InputImageType * );
+  InputImageType *
+  GetInput();
 
 protected:
   VTKImageExport();
   ~VTKImageExport() override = default;
-  void PrintSelf(std::ostream & os, Indent indent) const override;
+  void
+  PrintSelf( std::ostream & os, Indent indent ) const override;
 
   using InputImagePointer = typename InputImageType::Pointer;
   using InputRegionType = typename InputImageType::RegionType;
   using InputSizeType = typename InputRegionType::SizeType;
   using InputIndexType = typename InputRegionType::IndexType;
 
-  int * WholeExtentCallback() override;
+  int *
```

Comment:
This kind of blindness is the price to pay for automated consistency. But I am willing to do it in order to have good, automated consistency.

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

*** . AlwaysBreakAfterReturnType = All,  but we could go with "top level" or "None"

Reply:
The issue about whether to put return type definition on a separate line is primarily its length. An `int` or a `void` don't benefit from being put on a separate line, but [`typename MorphologicalContourInterpolator< TImage >::BoolSliceType::Pointer`](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/blob/4f4fba267b93af6af379d676160d871ee1fdd3d2/include/itkMorphologicalContourInterpolator.hxx#L259) does.

` clang-format` does not seem to offer the option we like, so we should stick with `All`, which is probably the closest to what we want and already part of ITK's recommended style.